test/mpi/manual/mpi_t/nem_fbox_fallback_to_queue_count.c

```diff
@@ -56,7 +57,7 @@ void blank_test()
  * Here, the sender posts all sends before the receiver has a chance to
  * acknowledge any of them; this should force the sender to fall_back to the
  * queue every time. */
-void send_first_test()
+static void send_first_test()
```

Comment:
should we make the argument list for this and following functions `void`?

Reply:
`void` is only required for the prototype, not static functions. But I don't really have a preference, so your pick.

Reply:
Added "void".